docs/hugo/content/reference/_index.md

```diff
@@ -16,265 +16,265 @@ grouped by the originating ARM service.
 
 ## Authorization
 
-| Resource       | ARM Version        | CRD Version                | Supported From | Sample                                                                                                                                           |
```

Comment:
I think that this file is autogenerated? So you need to fix the code that autogenerates it?

Also might need to actually make a note someplace in the file in a markdown comment (so it's not rendered) that: "THIS IS A GENERATED FILE... DO NOT MODIFY IT" or something.